### PR TITLE
feat: add @nodesecure/authors

### PR DESCRIPTION
**Pull request by Kawacrepe**
Related to https://github.com/NodeSecure/authors/issues/10

Also: 
- update deps
- remove .jest file

### CODE REVIEW

**Comment by Kawacrepe** [comment]
We may want to create a different issue about the TODO I put in the code.
This is mostly related to RC, we may want to give this opportunity to someone who wants to start contributing.

It consist in adding `flags` & `domainInformations` in RC and use em in scanner

**Comment by Kawacrepe** [comment]
We may want to create a different issue about the TODO I put in the code.
This is mostly related to RC, we may want to give this opportunity to someone who wants to start contributing.

It consist in adding `flags` & `domainInformations` in RC and use em in scanner

File: src/utils/warnings.js [hunk 1]
```diff
@@ -8,29 +9,28 @@ const kWarningsMessages = Object.freeze({
   iohook: getToken("warnings.keylogging")
 });
 const kPackages = new Set(Object.keys(kWarningsMessages));
-const kAuthors = new Set(["marak", "marak.squires@gmail.com"]);
+const KFlags = [{
```
Comment (fraxken) [inline src/utils/warnings.js:12]: `k` should always be in lowercase. `kFlaggedAuthors` ?

**Review by fraxken — CHANGES_REQUESTED** [review]

**Comment by fraxken** [comment]
This line should be updated: https://github.com/NodeSecure/scanner/blob/master/src/depWalker.js#L326

**Comment by Kawacrepe** [comment]
@fraxken What do you think about this kind of payload ?

```js
{
  warnings: {
    globals: globalsWarning,
    flaggedAuthors: xx.filter((el) => el.flagged === true))
  },
  authors,
}
```

It may be future-proof (I'm thinking about nodesecure/rc atm)

**Comment by fraxken** [comment]
@Kawacrepe do we really need to touch the global `warnings` key ? Because for me if the flag information is in the list of authors we do not need to repeat the information.

Then for me it's the responsability of tools to filter flagged authors (like in Scanner / UI / CI ..).

**Comment by Kawacrepe** [comment]
Oh okay, looks like I didn't understood one of your messages in the discord channel

**Comment by Kawacrepe** [comment]
Thank's to this suit of tests, Now, we know we need to release an author version that handle scenario when there is no authors, name & email in the given dependency

**Review by fraxken — APPROVED** [review]

**Comment by fraxken** [comment]
@all-contributors please add @Kawacrepe for code

**Pull request merged**